woodwork/table_accessor.py

```diff
@@ -88,8 +89,12 @@ def init(self,
                 Any other arguments provided will be ignored. Note that any changes made to the schema object after
                 initialization will propagate to the DataFrame. Similarly, to avoid unintended typing information changes,
                 the same schema object should not be shared between DataFrames.
+            validate (bool, optional): Whether parameter and data validation should occur. Defaults to True. Warning:
+                Should only be False when confident that parameters and data are valid since any resulting errors may
+                not be easily understood.
```

Comment:
Wording suggestion: `Should be set to False only when parameters and data are known to be valid. Any errors resulting from skipping validation with invalid inputs may not be easily understood.`

Reply:
Made these changes here and in Schema